downstream/modules/platform/proc-synchronizing-rpm-repositories-by-using-reposync.adoc

```diff
@@ -28,7 +31,7 @@ To perform a reposync you need a RHEL host that has access to the Internet. Afte
 
 .. If using `--newest-only` the repos downloaded will be ~14GB.
 
-. If you plan to use Red Hat Single Sign-On (RHSSO) sync these repositories.
+. If you plan to use {RHSSO}, sync these repositories:
```

Comment:
Curious about line 28 (reference to [RHEL 8] Reposync). Is this a standard for RHEL?